conversions/decimal_to_binary_recursion.py

```diff
@@ -1,25 +1,23 @@
-""" Convert decimal values to binary using recursion method """
-
-
-def bin_recursive(decimal: int) -> str:
+def binary_recursive(decimal: int) -> str:
     """
-    The funtion takes in a positive integer value
+    This takes in a positive integer value
     and returns its binary equivalent.
```

Comment:
```suggestion
    Take in a positive integer and return its binary equivalent.
```
I tend to use an imperative voice — Just tell the computer what it should do.

Reply:
ok